addons/sys_godmode/script_component.hpp

```diff
@@ -15,4 +15,20 @@
 #endif
 
 #include "\idi\acre\addons\main\script_macros.hpp"
-#include "script_acre_component_defines.hpp"
+
+
+// Group defines
+#define GODMODE_CURRENTCHANNEL   0
```

Comment:
API functions use these defines. That is why they are declared in addons/sys_godmode/script_acre_component_defines.hpp